src/syntax/FStar.Syntax.Visit.fsti

```diff
@@ -0,0 +1,9 @@
+module FStar.Syntax.Visit
+
+open FStar.Syntax.Syntax
+
+val visit_term : (term -> term) -> term -> term
```

Comment:
Some docs/comments here would be useful, especially if there are some parts of the syntax that are not covered here?

Reply:
Will do